.github/workflows/changelog.yml

```diff
@@ -0,0 +1,31 @@
+name: changelog
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  cliffy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ secrets.CHANGELOG_COMMITTER_APP_ID }}
+          private-key: ${{ secrets.CHANGELOG_COMMITTER_PRIVATE_KEY }}
```

Comment:
- [ ] Plug in secrets you own

I think this is required to allowlist a particular identity to commit this change (and bypass your branch protection settings).